File: dotnet/Lexy/Lexy.Compiler/Compiler/CSharp/BuiltInFunctions/SecondFunctionCall.cs
using Lexy.Compiler.Language.Expressions.Functions;
using Lexy.RunTime;

namespace Lexy.Compiler.Compiler.CSharp.BuiltInFunctions;

internal class SecondFunctionCall : SingleArgumentFunctionCall<SecondFunction>
{
    protected override string ClassName => nameof(BuiltInDateFunctions);
    protected override string MethodName => nameof(BuiltInDateFunctions.Second);
}

File: typescipt/src/compiler/JavaScript/builtInFunctions/secondFunctionCall.ts
import {SingleArgumentFunctionCall} from "./singleArgumentFunctionCall";
import {SecondFunction} from "../../../language/expressions/functions/secondFunction";

export class SecondFunctionCall extends SingleArgumentFunctionCall {
   protected override className = "BuiltInDateFunctions";
   protected override methodName = "second";

   constructor(functionNode: SecondFunction) {
      super(functionNode);
   }
}
